integration_tests/test_employment.py

```diff
@@ -45,3 +45,27 @@ def test_employed_have_income(tracked_live_populations, time_step):
     # All people in military group quarters are
     employed = pop[pop["employer_id"] != data_values.UNEMPLOYED_ID]
     assert (employed["income"] > 0).all()
+
+
+def test_movers_change_employment(sim):
```

Comment:
The testing framework is designed to have all the information you need without needing to take any more time-steps of the simulation. You should be able to do all of this by just looking at subsequent snapshots of the population state table stored in the `populations` fixture.

Reply:
I've extracted the changes I need for this into #128.

Reply:
@rmudambi I've now corrected this test to use the `populations` fixture.